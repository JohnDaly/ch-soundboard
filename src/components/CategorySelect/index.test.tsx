// External Dependencies
import { mount } from 'enzyme'
import enzymeToJSON from 'enzyme-to-json'
import 'jest-styled-components'
import React from 'react'

// Internal Dependencies
import { CategorySelect } from './index'

describe('CategorySelect', () => {
    it('should match the snapshot', () => {
        const mockedOnSelect = jest.fn()
        const wrapper = mount(
            <CategorySelect config={{}} onSelectCategory={mockedOnSelect} />
        )
        expect(enzymeToJSON(wrapper)).toMatchSnapshot()
    })
})
