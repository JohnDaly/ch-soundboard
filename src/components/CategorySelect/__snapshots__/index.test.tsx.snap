// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CategorySelect should match the snapshot 1`] = `
.c0 {
  font-size: 12px;
  margin-right: 0.5rem;
}

<Component
  config={Object {}}
  onSelectCategory={[Function]}
>
  <div>
    <CategoryLabel
      label="Category"
    >
      <Component
        className="c0"
        label="Category"
      >
        <span
          className="c0"
        >
          Category
        </span>
      </Component>
    </CategoryLabel>
    <select
      onChange={[Function]}
      value="All"
    >
      <CategorySelectOption
        category="All"
        key="category_All"
      >
        <option
          value="All"
        >
          All
        </option>
      </CategorySelectOption>
    </select>
  </div>
</Component>
`;
